myst.yml

```diff
@@ -0,0 +1,29 @@
+# See docs at: https://mystmd.org/guide/frontmatter
+version: 1
+project:
+  id: 7a62a86d-a893-4ab1-9473-b1a957f78902
+  title: GTSAM Docs
+  description: GTSAM is a library of C++ classes that implement smoothing and mapping (SAM) in robotics and vision, using factor graphs and Bayes networks as the underlying computing paradigm rather than sparse matrices.
+  github: https://github.com/borglab/gtsam
+  toc:
+    - file: README.md
+    - file: INSTALL.md
+    - title: Python Reference
+      children: 
+        - pattern: './doc/PythonReference/*'
```

Comment:
So, I think we want these notebooks- if possible, alongside the c++ source code in a `doc` folder (next to `tests`). Even though the example in code in these is python, I see that as incidental - the content (aside from the demo code) should be language agnostic: it is a about the math.